toqito/state_props/tests/test_is_abs_ppt.py

```diff
@@ -0,0 +1,79 @@
+"""Test cases for the `is_abs_ppt` function."""
+
+import numpy as np
+import pytest
+
+from toqito.state_props.is_abs_ppt import is_abs_ppt
+from toqito.states import bell, max_mixed
+
+
+@pytest.mark.parametrize(

```

Comment:
Can you please verify that your approach also successfully runs on those in the reference implementation, please?
http://qetlab.com/AbsPPTConstraints